reconcile/jira_permissions_validator.py

```diff
@@ -211,9 +211,14 @@ def validate_boards(
     jira_clients: dict[str, JiraClient] = {}
     for board in jira_boards:
         next_run_time = state.get(board.name, 0)
-        if not dry_run and time.time() <= next_run_time:
-            logging.debug(f"[{board.name}] Skipping board")
-            continue
+        if time.time() <= next_run_time:
+            if not dry_run:
+                # always skip for non-dry-run mode
+                continue
+            # dry-run mode
+            elif len(jira_boards) > 1:
+                # skip for MR checks if there are more than one board to test
+                continue
```

Comment:
do you want to log something here to know the check is skipped?

Reply:
Good idea. Added in b438c01